draco/analysis/powerspectrum.py

```diff
@@ -0,0 +1,90 @@
+"""Power spectrum estimation code.
```

Comment:
I reviewed 'powerspectrum.py' and did not find anything out of the ordinary.
I note the low comment-to-code ratio. ;)